### PR TITLE
Redir formula

### PR DESCRIPTION
**Pull request by seocam**
Redir is a port redirector.  It's functionally basically consists of
the ability to listen for TCP connections on a given port, and, when
it recieves a connection, to then connect to a given destination
address/port, and pass data between them.  It finds most of its
applications in traversing firewalls, but, of course, there are other
uses.

### CODE REVIEW

File: Library/Formula/redir.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Redir < Formula
+  homepage "http://sammy.net/~sammy/hacks/"
+  url "https://github.com/TracyWebTech/redir/archive/2.2.1-9.tar.gz"
+  version "2.2.1_9"
```
Comment (mikemcquaid) [inline Library/Formula/redir.rb]: Can probably skip this and jsut use the one from the URL

File: Library/Formula/redir.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  sha1 "84ae75104d79432bbc15f67e4dc2980e0912b2b6"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make"
+    system "install -d #{prefix}/bin #{man}/man1"
+    system "install redir #{prefix}/bin/redir"
+    system "install redir.man #{man}/man1/redir.1"
```
Comment (mikemcquaid) [inline Library/Formula/redir.rb]: No `make install`?

File: Library/Formula/redir.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/redir.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/redir.rb (hunk 1 above)
Comment (seocam) [inline Library/Formula/redir.rb]: It was installing using as version `9` instead of `2.2.1-9`.

File: Library/Formula/redir.rb (hunk 2 above)
Comment (seocam) [inline Library/Formula/redir.rb]: No, there is no make install in the original Makefile :(

File: Library/Formula/redir.rb (hunk 3 above)
Comment (seocam) [inline Library/Formula/redir.rb]: Redir runs as a daemon so I couldn't find a good way to test and then terminate the daemon. Do you have any good examples that would be similar to that?

File: Library/Formula/redir.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/redir.rb]: Ok, thanks.

File: Library/Formula/redir.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Formula/redir.rb]: Might be nice to submit a PR to add this to the Makefile?

File: Library/Formula/redir.rb (hunk 3 above)
Comment (mikemcquaid) [inline Library/Formula/redir.rb]: Even `--help` or `--version` would be better than nothing.

File: Library/Formula/redir.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+class Redir < Formula
+  homepage "http://sammy.net/~sammy/hacks/"
+  url "https://github.com/TracyWebTech/redir/archive/2.2.1-9.tar.gz"
+  version "2.2.1_9"
+  sha1 "84ae75104d79432bbc15f67e4dc2980e0912b2b6"
+
+  depends_on "cmake" => :build
```
Comment (mikemcquaid) [inline Library/Formula/redir.rb]: Hmm, are you sure this is needed?

Comment (seocam) [inline Library/Formula/redir.rb]: Actually it's not needed. I've missed it. I'll send an updated in a few minutes.

**Comment by seocam** [comment]
What about now? ;)

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `redir 2.2.1_9 (new formula)`.

**Pull request closed without merging**